src/bin/tikv-server.rs

```diff
@@ -602,7 +612,24 @@ fn run_raft_server(listener: TcpListener,
                           resolver,
                           snap_mgr)
         .unwrap();
-    start_server(svr, event_loop, engine);
+
+    let backup_path = {
+        let mut path = get_toml_string(config, "server.backup", Some(String::new()));
+        if path.is_empty() {
+            path = store_path.to_owned();
```

Comment:
the real db path is `store_path/db`, seem we can use `store_path/db_backup` directly.